tst/EnergyPlus/unit/FaultsManager.unit.cc

```diff
@@ -192,3 +200,66 @@ TEST_F( EnergyPlusFixture, FaultsManager_FaultFoulingAirFilters_CalFaultyFanAirF
 	Fan.deallocate();
 
 }
+
+TEST_F( EnergyPlusFixture, FaultsManager_FaultChillerSWTSensor_CalFaultChillerSWT )
```

Comment:
Mike, this function is applicable for all the chiller types that are involved in this fault feature, not just Chiller:Electric:Reformulated:EIR. I checked it when I applied it to different chiller types.